src/components/Book.jsx

```diff
@@ -36,14 +42,18 @@ const Book = ({ book }) => {
 
       <div className="progressBar">
         <CircularProgressbar
-          value={parseInt(book.completed, 30)}
-          text={`${book.completed}`}
+          value={parseInt(boook.completed, 30)}
+          // text={`${boook.completed}`}
```

Comment:
Great job @Simpleshaikh1 🤝

- [ ] Kindly remove commented code from your project as this is regarded as bad practice.